applications/NeuroZK.md

```diff
@@ -0,0 +1,266 @@
+
+Once the docker container is deployed, its workload will be protected by software based measures to prevent data leakage.
+
+It seamlessly integrates with the current cloud-native toolchain for developers, typically avoiding the need for code changes.
+ 
+Developers can monitor details such as resource utilization, and the app's health status through the Cyborg Connect application dashboard.
+
+![dashboard](https://github.com/Cyborg-Network/Grants-Program/assets/93442895/90f160c6-5cef-408b-8db2-c3e550fdb6a8)
+
+### ZK Snark proof verification 
+
+For this grant application, we will improvise our previous submission to make it live on Paseo and then proceed with developing a decentralized proof verification system to incorporate an on-chain ZK Snark system for verifying AI inference models executed in an off-chain environment. The main components will include the Cyborg AppChain (Deployed on Tanssi's Dancebox testnet), Cyborg Worker node (A custom node implementation to manage running offchain tasks and streaming proofs into pallets for verification), TEE Oracle, and a testing suite.
+
+The objective is to deploy a pre trained AI inference model in servers at different geographical locations based on user requirements. The Substrate blockchain will establish secure communication with worker nodes to issue commands for deploying, testing, and verifying model executions.
+
+We will define a new Substrate node configuration for the Cyborg worker nodes, which users can rent on-demand from the pool of available worker nodes.
+
+
+### Ecosystem Fit
+
+Cyborg Network enhances the Polkadot ecosystem with decentralized edge computing, addressing the increasing demand for AI and IoT applications. We focus on Web2 application providers, opening doors for new users to join the Polkadot ecosystem and support its growth. We are also integrating enterprises that operate edge data centers into the Polkadot ecosystem. Some of our potential partners include [Edge Centres](https://edgecentres.com/) , [Deep Green](https://deepgreen.energy/) , [Heata](https://www.heata.co/) etc.
+
+Though Cyborg and the [Phala Network](https://phala.network/) share aspirations of decentralized computing within the Polkadot/Kusama environment, our technical paths and business aspirations are distinct. Phala Network provides off-chain compute infrastructure for smart contract-based applications by enabling users to integrate 'phat' contracts with their existing smart contract logic. While Cyborg is focused on creating utility for web2 applications.
+
+Another related project appears to be [Wetee](https://github.com/wetee-dao), which employs a comparable technical architecture utilizing TEEs to safeguard user privacy. However, their specific market focus is currently untraceable. We are developing products for a validated market that has already attracted interest from various AI applications and 50 edge data center providers to join the network once it goes live.
```

Comment:
I apologize for failing to update that information from our previous grant application. Since we have shifted our focus to launching as an AI inference-focused chain, data center interactions are no longer our primary emphasis. Cyborg Network will launch as a parachain that will govern numerous plug-and-play edge servers (Cyborg worker nodes), which are custom-built standard spec edge servers sold by our organization. The vision is to strategically distribute this infrastructure into commercial establishments, like convenience stores, to execute AI inference tasks generated from nearby data sources, such as data aggregated from sensors in smart cities.

However, some interested parties have expressed a desire to engage with us by providing a cloud backup solution for these worker nodes to ensure data availability in the system. I will update the document with a table that clearly mentions the name of the company, the scope of interaction, and the current status soon.